kie-wb-tests/kie-wb-tests-gui/src/main/java/org/kie/wb/selenium/model/persps/ProjectLibraryPerspective.java

```diff
@@ -15,53 +15,35 @@
  */
 package org.kie.wb.selenium.model.persps;
 
-import org.kie.wb.selenium.model.persps.authoring.ImportExampleModal;
-import org.kie.wb.selenium.util.Repository;
 import org.kie.wb.selenium.util.Waits;
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
 
 public class ProjectLibraryPerspective extends AbstractPerspective {
 
-    //<div class="full" data-field="emptyLibrary">
-    private static final By PROJECT_LIBRARY_HOLDER = By.xpath( "//div[@data-field='emptyLibrary']" );
+    private static final String OU_REPO_BREADCRUMB_XPATH_SELECTOR = "//a[text()='myteam (myrepo)']";
```

Comment:
@jhrcek I agree, we could explore more the `@FindByJQuery` annotation. 

@bsremac After this PR get merged, maybe it's a good thing to improve. :)